src/test/java/org/broadinstitute/hellbender/utils/python/PythonEnvironmentIntegrationTest.java

```diff
@@ -43,9 +42,9 @@ public Object[][] getDataPackagePresent() {
     @DataProvider(name="dataPackageMKLEnabled")
     public Object[][] getDataPackageMKLEnabled() {
         return new Object[][] {
-                { "numpy",          "numpy.__config__.get_info('blas_mkl_info') != {} and numpy.__config__.get_info('lapack_mkl_info') != {}" },
-                { "theano",         "'-lmkl_rt' in theano.config.blas.ldflags" },
-                { "tensorflow",     "tensorflow.pywrap_tensorflow.IsMklEnabled()" }
+                // { "numpy",          "numpy.__config__.get_info('blas_mkl_info') != {} and numpy.__config__.get_info('lapack_mkl_info') != {}" },  TODO this check might need to be done differently for conda-forge numpy 1.26.2, we remove it for now
```

Comment:
Is this a real TODO?

Reply:
Yes, TBH I'm not sure how to check for MKL versions now. But maybe it doesn't matter if runtime seems OK. I'll leave this here with an updated note just in case.